internal/pkg/instrumentation/bpf/go.opentelemetry.io/otel/traceglobal/bpf/probe.bpf.c

```diff
@@ -162,6 +169,35 @@ int uprobe_SetAttributes(struct pt_regs *ctx) {
     return 0;
 }
 
+// This instrumentation attaches uprobe to the following function:
+// func (nonRecordingSpan) SetName(string)
+SEC("uprobe/SetName")
+int uprobe_SetName(struct pt_regs *ctx) {
+    void *non_recording_span_ptr = get_argument(ctx, 1);
+    struct otel_span_t *span = bpf_map_lookup_elem(&active_spans_by_span_ptr, &non_recording_span_ptr);
+    if (span == NULL) {
+        return 0;
+    }
+
+    void *span_name_ptr = get_argument(ctx, 4);
```

Comment:
I think it should be `get_argument(ctx, 2);`
```suggestion
    void *span_name_ptr = get_argument(ctx, 2);
```

Reply:
correct -  updated 😄